libraries/wallet/wallet.cpp

```diff
@@ -808,10 +808,24 @@ class wallet_api_impl
          //
          // http://en.wikipedia.org/wiki/Most_vexing_parse
          //
-         fc::ofstream outfile{ fc::path( wallet_filename ) };
+         fc::ofstream outfile{ fc::path( wallet_filename + ".tmp" ) };
          outfile.write( data.c_str(), data.length() );
          outfile.flush();
          outfile.close();
+
+         std::string wallet_file_content;
+         fc::read_file_contents(wallet_filename, wallet_file_content);
```

Comment:
Here you are not reading from the saved file. Why not use a local variable (E.G. `tmp_file_name`)?

Reply:
you're absolutely right ! made changes already ! Thank you !